libs/platform/src/lib/components/form/checkbox/checkbox.component.html

```diff
@@ -0,0 +1,18 @@
+<div fd-form-item>
+    <fd-checkbox
+        [inputId]="id"
+        [name]="name"
+        [disabled]="disabled"
+        [label]="label"
+        [attr.aria-label]="ariaLabel || null"
+        [attr.aria-labelledby]="ariaLabelledby"
+        [tristate]="tristate"
+        [tristateSelectable]="tristateSelectable"
+        [state]="stateStatus"
```

Comment:
For the state please follow similar path like other component, you dont need extra setter/getter.  This where we have in base input method that checks error and just mark component if in error state or not. 


If we want to support in general all statuses (even i dont see usecase why we would show info status, or success - maybe to mark some questionaries "yes you did good", "all is wrong", "we are not sure"), we better move state to the parent class. 

state needs to be driven by FormAPI.

Reply:
i was thinking checkbox group may require some way to update status of individual checkbox. so while implementing, this setStatus will be used. have removed this. Now Directly getting state from formApi status.

Reply:
I see. Can you find out from @manjunathanagaraj  , when we have such usecase where will have checkboxgroup if we will mark individual checkbox? I am not sure, then you need to provide a message. 

If we are talkg about isBinary, yes/no, then it make sense but again it should follow regular form/error handling logic like we have it. What do you think ?

Reply:
Now I have kept it where state is read from form Api Status.

Reply:
taking state value as form status.